hardware/taito/include/taitoScreen.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef _TAITO_SCREEN_HPP_
+#define _TAITO_SCREEN_HPP_
+
+extern "C"
+{
+#include "cpu.h"
+#include "screen_timer.h"
+#include "taito_struct.h"
+}
+
+#include "hw_lib_imports.h"
+
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_image.h>
+#include <SDL2/SDL_timer.h>
```

Comment:
Do we need to include `SDL_timer.h`?  I don't think we're using it in this thread.